Rakefile

```diff
@@ -333,6 +333,8 @@ namespace :plugin do
 
           source_files = Dir.glob(File.join("Bugsnag", "**", "*.{c,h,mm,cpp,m}"))
                             .map(&File.method(:realpath))
+                              # For some reason this file is included already and including it again causes an error that "multiple commands produce" it.
```

Comment:
I'm keen to know what's actually causing this, i'm worried that if we fix it here, then it could change elsewhere and break. Please get in touch with the cocoa team and explain what you know so far, maybe they can fix it on their end.

Reply:
This was caused by [commit 73e8426](https://github.com/bugsnag/bugsnag-cocoa/commit/73e84267dc5a4d69b4b73cf830d0dfa0910b23df).